client/src/components/_common/Button/Button.jsx

```diff
@@ -53,18 +55,31 @@ const Button = ({
     buttonSizeClass = styles[`width-${size}`];
   }
 
+  let loadingIcon = null;
+  if (isLoading) {
+    loadingIcon = (
+      <LoadingSpinner
+        placement={styles['over-text']}
+        className={styles['loading-over-button']}
+      />
+    );
+  }
+
   return (
     <button
       className={`c-button ${buttonTypeClass} ${buttonSizeClass}`}
-      disabled={disabled}
+      disabled={disabled || isLoading}
       type={attr}
       onClick={onclick}
     >
+      {loadingIcon}
```

Comment:
No change necessary, but another way to do this:
```suggestion
      {isLoading && (
        <LoadingSpinner
          placement={styles['over-text']}
          className={styles['loading-over-button']}
        />
      )}
```

Reply:
Ah, yes, forgot this is done via this approach in other spots in the codebase - will change!